### PR TITLE
Update tv.zap & matter-devices.xml to align them with spec

### PR DESCRIPTION
**Pull request by lazarkov**
#### Problem
- [balloob](https://github.com/balloob) pointed out that matter devices is not per spec.
- @chrisdecenzo pointed out that endpoints 4 & 5 are not needed in tv.zap & are not per spec.
- @chrisdecenzo pointed out that endpoint 3 must implement keypad input (0x509), app launcher (0x50c), app basic (0x50d) and account login (0x50e) in order to pull that cluster server into the build

#### Change overview
see above

#### Issue(s) fixed:
[19341](https://github.com/project-chip/connectedhomeip/issues/19341)

#### Testing
Run `./scripts/run_in_build_env.sh \  
                  "./scripts/tests/run_test_suite.py \
                     --chip-tool ./out/debug/standalone/chip-tool \
                     run \
                     --iterations 1 \
                     --all-clusters-app ./out/debug/standalone/chip-all-clusters-app \
                     --tv-app ./out/debug/standalone/chip-tv-app \
                  "`

### CODE REVIEW

**Review by chrisdecenzo — APPROVED** [review]
looks good but should only have binding cluster on ep0

**Review by woody-apple — APPROVED** [review]

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml [hunk 1]
```diff
@@ -1990,6 +1990,44 @@ limitations under the License.
...
+                <requireAttribute>PARTS_LIST</requireAttribute>
+            </include>
+            <include cluster="Binding" client="true" server="false" clientLocked="false" serverLocked="true">
+                <requireAttribute>BINDING</requireAttribute>
+            </include>
+            <include cluster="Media Playback" client="false" server="true" clientLocked="false" serverLocked="true"></include>
+            <include cluster="Keypad Input" client="false" server="true" clientLocked="false" serverLocked="true"></include>
+            <include cluster="Application Launcher" client="false" server="true" clientLocked="false" serverLocked="false"></include>
+            <include cluster="Media Input" client="false" server="true" clientLocked="false" serverLocked="false"></include>
+            <include cluster="On/Off" client="false" server="true" clientLocked="false" serverLocked="false"></include>
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:2021]: Shouldn't ServerLocked be true for On/Off?  It's a required cluster for this device.

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml [hunk 2]
```diff
@@ -1990,6 +1990,44 @@ limitations under the License.
...
+                <requireCommand>IdentifyQuery</requireCommand>
+                <requireCommand>TriggerEffect</requireCommand>
+            </include>
+            <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true">
+                <requireAttribute>DEVICE_TYPE_LIST</requireAttribute>
+                <requireAttribute>SERVER_LIST</requireAttribute>
+                <requireAttribute>CLIENT_LIST</requireAttribute>
+                <requireAttribute>PARTS_LIST</requireAttribute>
+            </include>
+            <include cluster="Binding" client="true" server="false" clientLocked="false" serverLocked="true">
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:2014]: Why is this including the binding client?

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml [hunk 3]
```diff
@@ -1990,6 +1990,44 @@ limitations under the License.
...
     </deviceType>
+
+    <deviceType>
+        <name>MA-basic-videoplayer</name>
+        <domain>CHIP</domain>
+        <typeName>Matter Basic Video Player</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x0028</deviceId>
+        <clusters lockOthers="true">
+            <include cluster="Identify" client="false" server="true" clientLocked="true" serverLocked="true">
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:2001]: Why is this forcing inclusion of Identify?  I don't see Identify in the device spec for basic video player.

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml [hunk 4]
```diff
@@ -1990,6 +1990,44 @@ limitations under the License.
...
+                <requireAttribute>SERVER_LIST</requireAttribute>
+                <requireAttribute>CLIENT_LIST</requireAttribute>
+                <requireAttribute>PARTS_LIST</requireAttribute>
+            </include>
+            <include cluster="Binding" client="true" server="false" clientLocked="false" serverLocked="true">
+                <requireAttribute>BINDING</requireAttribute>
+            </include>
+            <include cluster="Media Playback" client="false" server="true" clientLocked="false" serverLocked="true"></include>
+            <include cluster="Keypad Input" client="false" server="true" clientLocked="false" serverLocked="true"></include>
+            <include cluster="Application Launcher" client="false" server="true" clientLocked="false" serverLocked="false"></include>
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:2019]: Application Launcher is not in the basic video player device spec... Neither is Account Login.

Comment (lazarkov) [reply src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:2019]: Addressed in: https://github.com/project-chip/connectedhomeip/pull/21045

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml (hunk 3 above)
Comment (lazarkov) [reply src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:2001]: Addressed in: https://github.com/project-chip/connectedhomeip/pull/21045

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml (hunk 2 above)
Comment (lazarkov) [reply src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:2014]: Addressed in: https://github.com/project-chip/connectedhomeip/pull/21045

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml (hunk 1 above)
Comment (lazarkov) [reply src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:2021]: Addressed in: https://github.com/project-chip/connectedhomeip/pull/21045

**Pull request merged**